libs/game/sprite.ts

```diff
@@ -376,11 +376,11 @@ class Sprite extends sprites.BaseSprite {
         // If it's just a small change to the hitbox on one axis,
         // don't change the dimensions to avoid random clipping
         this._hitbox = newHitBox;
-        if (xDiff <= Fx.twoFx8) {
+        if (xDiff <= Fx.mul(Fx.twoFx8, this._sx)) {
```

Comment:
Hmm not certain I follow the change here; if scale is 2 then we will avoid updating the hitbox for differences up to 4px? It's an existing hack for minor position differences so leaving it at just 2 px seems preferable to me -- any more than that and the hitbox not changing will start to be noticeable.

Reply:
I made this change so that the behavior would be consistent at different scales, but I can see how it would be better to stick with the small visual diff. I'll back it out.